runtime/etc/mli/mli_client_runtime.c

```diff
@@ -52,6 +55,14 @@ void chpl_mli_client_deinit(struct chpl_mli_context* client) {
   return;
 }
 
+static void spawn_server() {
+  char command[256] = "./";
+  strcat(command, mli_servername);
+  strcat(command, " -nl 1");
+
+  printf("executing %s as a subprocess\n", command);
```

Comment:
Can we wrap this printf up in a `#ifdef CHPL_MLI_DEBUG_PRINT` so that it only displays when debug codegen is set to on?